### PR TITLE
twty 0.0.11 (new formula)

### PR DESCRIPTION
**Pull request by umireon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
A command-line twitter client written in golang.

I will work on other PRs after this PR is merged. I just need time.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
lgtm

File: Formula/twty.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  desc "Command-line twitter client written in golang"
+  homepage "https://mattn.kaoriya.net/"
+  url "https://github.com/mattn/twty/archive/refs/tags/v0.0.11.tar.gz"
+  sha256 "d1ee544ff31a9a9488ff759da587baf927ab7c31b191b4b5bc010f36ecfb8188"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
```
Comment (chenrui333) [inline Formula/twty.rb:11]: Actually we can update the ldflags to shrink the binary
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment (umireon) [reply Formula/twty.rb]: Thanks!

**Comment by umireon** [comment]
Ldflags updated and test passed.
Review ready

**Review by carlocab — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]
Thanks for the update!

**Comment by umireon** [comment]
Thanks for merging!

**Pull request closed without merging**